### ISSUE TITLE
[a11y-defect-2]: VoiceOver cursor does not match programmatic focus in Safari

### ISSUE THREAD

**Issue by maggiewachs** [department-of-veterans-affairs/va.gov-team#131615]
### Point of contact

Maggie Wachs

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

This issue was discovered during staging review for the VA Form 21P-534EZ. 

From the original ticket (https://github.com/department-of-veterans-affairs/va.gov-team/issues/124516): 

> In Safari, with Voiceover, programmatic focus is placed correctly on the Progress Bar - Segmented, however Voiceover cursor remains on the Continue or Back buttons, so Voiceover users aren't given an indication that they are on a new page, which may be confusing.

> Note: Safari cursor is handled correctly when the step is a single entry step - e.g. "Do you have a dependent child to add?" - Focus and Voiceover cursor/focus are both on the heading.

### Reproduction steps

1. In Safari, open the [534EZ in staging](https://staging.va.gov/family-and-caregiver-benefits/survivor-compensation/apply-for-dic-survivors-pension-accrued-benefits-form-21p-534ez/) and log in with test credentials (username: vets.gov.user+44@gmail.com, password: 191SsNrLgPv5)
2. Start VoiceOver and fill out the first page of Step 1 and hit Continue
3. Note where focus is assigned when the next question loads. You may need to fill out a few questions to trigger the bug.

### Proposed solution or next steps

This may be due to programmatically applying focus to an element, the progress bar, that is not assigned an ARIA role.

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [ ] Axe-core
- [x] Screenreader | assistive tech & device support
- [x] Keyboard
- [ ] Voice command
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [x] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else
